src/sources/kubernetes_logs/mod.rs

```diff
@@ -78,30 +82,55 @@ const FILE_KEY: &str = "file";
 const SELF_NODE_NAME_ENV_KEY: &str = "VECTOR_SELF_NODE_NAME";
 
 /// Configuration for the `kubernetes_logs` source.
+#[serde_as]
 #[configurable_component(source("kubernetes_logs"))]
 #[derive(Clone, Debug)]
 #[serde(deny_unknown_fields, default)]
 pub struct Config {
-    /// Specifies the label selector to filter `Pod`s with, to be used in addition to the built-in `exclude` filter.
+    /// Specifies the label selector to filter `Pod`s with, to be used in addition to the built-in
```

Comment:
Do you think there would be any value in adding some links to k8s documentation on the key words highlighted in this config struct? (Pod here, Namespace, Node below, etc.)

Reply:
I'd say no? They're very basic fundamentals of the Kubernetes API. Arguably, it wouldn't be unreasonable (I think) to remove the formatting around all those terms.

Reply:
Well, I guess at the same time - the install docs do link out to kube docs for `Node` 🤷 I'll drop them in there, it's easy enough.